README.md

```diff
@@ -1,69 +1,30 @@
+- For a remote DB, use [Mongo Atlas](https://www.mongodb.com/atlas/database). You can also use [Compass UI](https://www.mongodb.com/try/download/compass), but isn't strictly necessary. 
 
+In your MongoDB, add a database named `respond-dev`. Then also add a user for that database. In Mongo Atlas this is done by clicking on "Database Access" from the left side bar navigation. Then click the "Add new database user" and give them read/write permission. This username/password is what you'll use in your connection string below. 
 
-### Add Google auth configuration
-- Setup an OAuth 2.0 Client web app with Google (See https://support.google.com/cloud/answer/6158849?hl=en#zippy=%2Cweb-applications).
-- Add authentication information to `.env.local`:
-```
-GOOGLE_ID=<client-id>.apps.googleusercontent.com
-GOOGLE_SECRET=<client-secret>
-```
-- Add hostname `https://localhost:3000` to your Google app's Authorized Javascript Origins
-
-## Start Development Server
-
-### App startup
-In a terminal window:
-```
-yarn install
-yarn dev
-```
-
-Most of the site should auto-compile and update when the source file is saved. One exception seems to be the socket server (code initialized by `/api/socket-keepalive`), which needs you to Ctrl-C the server and start `yarn dev` again.
-
-As updates are made, the site will re-compile on demand (page load). You may notice a delay when navigating between pages.
-
-### SSL Proxy
-There are parts of the application (specifically Google authentication, possibly future geolocation) that don't work on an insecure (http://) site. To get into a secure context, we'll use an SSL proxy. HTTPS traffic on port 9001 is forwarded to an HTTP server on port 3000.
-
-In another terminal window, start the proxy:
-```bash
-npx local-ssl-proxy --target 3000
-```
-
-### Connect to app
-Open a web browser to https://localhost:9001
-
-
-## Seed Data
+#### Add Seed Data to MongoDB
 The app is multi-tenant, and matches hostnames to organizations stored in the database. To be able to use the application, at least one organization must be created in the database. Below are two documents that should allow you to get up and running.
 
-The application does have integration with the organization's membership database (currently D4H). In order to set this up you will need an API key from your member database admin.
+The application also has integration with D4H, so you will need an API key, either generated yourself or from your member database admin.
+**Note that a D4H API key is equivalent to your password: it provides full access to personal information for every KCSARA member - even the bits that are marked ‘private’. Make sure you don't make this key public (e.g. by checking it in).**
+Try to get a key yourself:
+- Go to https://kcsara.d4h.org/members/api_settings
```

Comment:
Please don't put this KCSARA specific URL In the public repo

Reply:
Switched this to use navigation instructions instead.

Reply:
I wasn't able to authenticate without both http://localhost:3000 and http://locahost as allowed domains. Leaving out http://localhost made the Google Auth window show up entirely blank every time I tried it, which makes me think there's something else weird going on. 
I haven't gotten this working with the proxy, so I don't know the proxy needs both https URLs added.